packages/clay-core/src/picker/Option.tsx

```diff
@@ -63,6 +63,11 @@ type Props = {
 	 */
 	index?: number;
 
+	/**
+	 * Flag to indicate that the option has href.
+	 */
+	href?: string | boolean | undefined;
```

Comment:
I think we don't need to add `boolean` here the same for `undefined` when adding that the property is optional (`href?: string;`) it is already marked as `undefined`. I believe that `boolean` was added here to handle the conditional, so I think it is not necessary because you can already have it with `undefined` or `string` it is enough and also it would not make much sense to have something like `href={true}` which would do nothing and having `href={false}` would be the equivalent of not passing the `href` which is marked as undefined.